test/server/compiler-test.js

```diff
@@ -971,4 +971,27 @@ describe('Compiler', function () {
             });
         });
     });
+
+    describe('Regression', function () {
+        it('Incorrect callsite line in error report on node v0.10.41 (GH-599)', function () {
+            this.timeout(5000);
+
+            var src         = 'test/server/data/regression/gh-599/testfile.js';
+            var compiler    = new Compiler([src]);
+
+            return compiler
+                .getTests()
+                .then(function (tests) {
+                    var test = tests[0];
+
+                    return test.fn(testRunMock);
+                })
+                .then(function () {
+                    throw 'Promise rejection expected';
+                })
+                .catch(function (err) {
+                    expect(err.errMsg || err).eql('Error: Error on line - 19');
```

Comment:
If the ["test"](https://github.com/DevExpress/testcafe/pull/677/files/aaf680fb3c582ecf0b55886e831ffd7bce156ddf#diff-f0b6a19b2ced6d03e927023709e31d79R987) was successful , this [line](https://github.com/DevExpress/testcafe/pull/677/files/aaf680fb3c582ecf0b55886e831ffd7bce156ddf#diff-f0b6a19b2ced6d03e927023709e31d79R990) leads to `err` object, which has no `errMsg` property , and doesn't matter that we are throwing [here](https://github.com/DevExpress/testcafe/pull/677/files/aaf680fb3c582ecf0b55886e831ffd7bce156ddf#diff-f0b6a19b2ced6d03e927023709e31d79R990) -  a `string` object or an `error`